src/test/java/com/puppycrawl/tools/checkstyle/internal/XpathRegressionTest.java

```diff
@@ -155,6 +155,7 @@
             "SuperClone",
             "SuperFinalize",
             "SuppressWarnings",
+            "SuppressWarningsHolder",
```

Comment:
This should be moved to something else as the check will never have xpath support. It doesn't create violations.

I would create a new field for it that specifically says they will not have xpath support ever, and give a reason why for this check.

Reply:
@rnveach Thanks, You are right
Once you create the new field I will rebase and move it to the new field

Reply:
@AmrDeveloper I was referring to as if I was working the PR myself. You should make the changes as part of your PR.

Reply:
As part of this PR, you need to update UT to skip such module at all.